client/components/empty-content/index.js

```diff
@@ -0,0 +1,185 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import { Component } from '@wordpress/element';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import { startsWith } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { H } from 'layout/section';
+import ImageAsset from 'components/image-asset';
+
+/**
+ * A component to be used when there is no data to show.
+ * It can be used as an opportunity to provide explanation or guidance to help a user progress.
+ */
+class EmptyContent extends Component {
+	renderIllustration() {
+		const { illustrationWidth, illustrationHeight } = this.props;
+		let illustrationSrc = this.props.illustration;
+		let ImageComponent = 'img';
+		if ( startsWith( illustrationSrc, '/' ) ) {
```

Comment:
Can we move this logic to `<ImageAsset />` ? If we alway use it, instead of `<img/>` we won't have to replicate this bit of code